package.json

```diff
@@ -128,55 +120,35 @@
     "lodash": "^2.4.2",
     "lscache": "~1.0.5",
     "lz-string": "^1.3.3",
-    "mailchimp-api-v3": "^1.4.1",
     "marked": "~0.3.0",
     "method-override": "^2.3.7",
     "moment": "~2.19.0",
     "moment-timezone": "~0.5.21",
-    "mongodb": "^2.0.28",
-    "mongoose": "^4.5.3",
-    "mongoose-cache": "github:nwinter/mongoose-cache#master",
     "morgan": "^1.7.0",
-    "mysql": "^2.11.1",
     "nanoscroller": "~0.8.0",
-    "node-force-domain": "~0.1.0",
-    "node-statsd": "^0.1.1",
     "npm-modernizr": "~2.8.3",
     "parse-full-name": "^1.2.3",
-    "passport": "0.1.x",
-    "passport-local": "0.1.x",
-    "paypal-rest-sdk": "^1.7.1",
-    "plyr": "^3.5.4",
     "promise-polyfill": "^5.2.1",
     "random-seed": "^0.3.0",
-    "request": "^2.75.0",
     "serve-favicon": "^2.3.2",
     "sortablejs": "^1.10.0-rc2",
-    "stream-buffers": "0.2.x",
     "string_score": "^0.1.20",
-    "stripe": "~2.9.0",
     "style-loader": "^0.13.1",
-    "swagger-ui-express": "^1.0.0",
-    "tmpl": "github:codecombat/JavaScript-Templates#remove-load",
     "treema": "git+https://github.com/codecombat/treema.git",
     "tv4": "~1.0.16",
     "underscore.string": "~2.3.3",
     "unibabel": "^2.1.8",
     "usa-states": "0.0.5",
     "v-tooltip": "^2.0.2",
     "vue": "^2.6.10",
-    "vue-codemirror": "^4.0.6",
     "vue-meta": "^1.6.0",
     "vue-moment": "^4.0.0",
     "vue-router": "^3.0.2",
     "vue-youtube": "^1.3.5",
-    "vuedraggable": "^2.21.0",
     "vuex": "^2.4.0",
     "vuex-router-sync": "^5.0.0",
     "webpack": "git://github.com/codecombat/webpack.git#webpack-3",
-    "webpack-shell-plugin": "^0.5.0",
-    "winston": "0.6.x",
-    "yamljs": "^0.2.8"
+    "webpack-shell-plugin": "^0.5.0"
```

Comment:
do we remove so many packages in frontend since we only use them in server end?

Reply:
Yep that's right! 👍